src/nupic/regions/SPRegion.py

```diff
@@ -159,29 +159,25 @@ def getConstraints(arg):
       description='Total number of columns (coincidences).',
       accessMode='Read',
       dataType='UInt32',
-      count=1,
-      constraints=''),
+      count=1 ),
 
     inputWidth=dict(
       description='Size of inputs to the SP.',
       accessMode='Read',
       dataType='UInt32',
-      count=1,
-      constraints=''),
+      count=1 ),
 
     spInputNonZeros=dict(
       description='The indices of the non-zero inputs to the spatial pooler',
       accessMode='Read',
       dataType='UInt32',
-      count=0,
-      constraints=''),
```

Comment:
@go-bears This video might help you understand git, Github, and the pull request model better:

[![Yc3PKaT1knU](http://img.youtube.com/vi/Yc3PKaT1knU/0.jpg)](http://www.youtube.com/watch?v=Yc3PKaT1knU)